### ISSUE TITLE
how can i add only single key all keyboard key   hot key list

### ISSUE THREAD

**Issue by ankitdifferenzsystem** [suysoftware/hotkey_system#1]

**Comment by sezeryavuz** [comment]
Unfortunately, the system does not allow recording a hotkey without using a modifier key. Typically, hotkey combinations require at least one modifier key (like Ctrl, Alt, cmd or Shift ) in conjunction with another key to prevent conflicts with regular keyboard operations.

**Comment by ankitdifferenzsystem** [comment]
hello @suysoftware  can i identify  any key press  evnet in background without using  modifier like  modifier key is default  upend  in  any button press event.

**Comment by sezeryavuz** [comment]
I didnt understand clearly what you want to say. Can you explain with more detail? @ankitdifferenzsystem

**Comment by ankitdifferenzsystem** [comment]
Hello @suysoftware , I'm wondering if there's a way to append the modifier key as default (without pressing it) along with other hotkeys in native code. Is this possible?

**Comment by sezeryavuz** [comment]
I don't think so. Because configuration should be   modifier key + key code  or 2x,3x modifier key + key code
